### ISSUE TITLE
increment_build_number_in_plist fails

### ISSUE THREAD

**Issue by eligat** [SiarheiFedartsou/fastlane-plugin-versioning#43]
In my project there are multiple targets, and even though action `get_build_number_from_plist(target: target, plist_build_setting_support: true)` retrieves correct build number, 
`increment_build_number_in_plist(target: target, plist_build_setting_support: true)`
fails with error:
```
ArgumentError: [!] wrong number of arguments (given 0, expected 1)
  /usr/local/lib/ruby/gems/2.4.0/gems/xcodeproj-1.13.0/lib/xcodeproj/project.rb:654:in `build_settings'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-plugin-versioning-0.4.0/lib/fastlane/plugin/versioning/actions/increment_build_number_in_xcodeproj.rb:62:in `set_build_number_using_target'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-plugin-versioning-0.4.0/lib/fastlane/plugin/versioning/actions/increment_build_number_in_xcodeproj.rb:23:in `run'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-plugin-versioning-0.4.0/lib/fastlane/plugin/versioning/actions/increment_build_number_in_plist.rb:25:in `run'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:256:in `block (2 levels) in execute_action'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:235:in `block in execute_action'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:227:in `chdir'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
  Fastfile:77:in `block (2 levels) in parsing_binding'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/lane.rb:33:in `call'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
  /usr/local/lib/ruby/gems/2.4.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
  /usr/local/lib/ruby/gems/2.4.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
  /usr/local/lib/ruby/gems/2.4.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
  /usr/local/lib/ruby/gems/2.4.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/commands_generator.rb:349:in `run'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
  /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.135.2/bin/fastlane:23:in `<top (required)>'
  /usr/local/bin/fastlane:23:in `load'
  /usr/local/bin/fastlane:23:in `<top (required)>'
```

**Comment by rfischer** [comment]
I had a similar problem with the action `increment_build_number_in_xcodeproj`. Using the unreleased version 0.4.2 solved it for me. See also https://github.com/SiarheiFedartsou/fastlane-plugin-versioning/issues/41

**Comment by jdouglas-nz** [comment]
sorry guys. I released 0.4.2 just now - please update and see if that helps!

**Comment by eligat** [comment]
Great! Version 0.4.2 fixes the issue. Thanks!
